packages/webapp/pages/onboarding.tsx

```diff
@@ -390,6 +391,7 @@ export function OnboardPage(): ReactElement {
       <PixelTracking />
       <GtagTracking />
       <TwitterTracking />
+      <RedditTracking />
```

Comment:
We're getting a few of these now, maybe we could bundle these scripts into one file?